Added section about max-input-vars (#3591)
magento/devdocs: guides/v2.1/install-gde/trouble/php/tshoot_php-set.md

	*	nginx (both CentOS and Ubuntu): `service nginx restart`
8.	Try the installation again.

### max-input-vars error due to large forms

> Some configurations with a high number of storeviews, products, attributes, or options may generate forms that exceed a PHP limit.

Reviewer: ```suggestion
Configurations with a high number of storeviews, products, attributes, or options can generate forms that exceed the preset PHP limit.
```

> If the number of values being sent surpasses the `max-input-vars` limit set within `php.ini` (default is 1000), some data will not be transferred. This limit may manifest by some values not being updated in the database. 

Reviewer: ```suggestion
If the number of values sent surpasses the `max-input-vars` limit set within `php.ini` (default is 1000), the remaining data is not transferred and those database values do not get updated.
```

> There may also be a warning in the PHP log:

Reviewer: ```suggestion
When this occurs, a warning appears in the PHP log:
```
Make this sentence less ambiguous.

More information about [asp_tags](http://php.net/manual/en/ini.core.php#ini.asp-tags){:target="_blank"}.

> Edit `php.ini` and set `asp_tags = off`. For more information, see [Required PHP settings][]

Reviewer 2: Add a period at the end of this sentence.

Author: Done.
